CONTRIBUTING.md

```diff
@@ -45,4 +45,11 @@ Every additional section will slow down the prompt a little bit. If your section
 * **Good:** check if command exists, check the value of environment variable
 * **Bad:** network requests, reading large files, etc
 
+### Documentation
+
+When updating documentation for your section, make sure the markdown document is being properly rendered by Github. Specifically, the following common pitfalls have already been discovered:
+
+- Empty inline code block `` will only be rendered if you put at least one space inside, like so: \` \` => ` `
```

Comment:
And ``` `` ``` 

Type 3 backticks, space, then 2, space and then 3 again for closing: 

![image](https://user-images.githubusercontent.com/3459374/35056809-954b0c18-fbbb-11e7-9342-52e84b88d71b.png)